src/encoding.rs

```diff
@@ -0,0 +1,87 @@
+use data_encoding::{Encoding, Specification};
+use lazy_static::lazy_static;
+
+fn make_encoding(symbols: &str, padding: Option<char>) -> Encoding {
```

Comment:
Recently https://github.com/multiformats/rust-multibase/pull/16 was merged.

There we use https://docs.rs/data-encoding-macro/0.1.7/data_encoding_macro/ for custom encodings.

See https://github.com/multiformats/rust-multibase/blob/4220b42bd422b940d46fd05589c2672e861f63a1/src/base.rs#L100-L103 for an example.

I think it would make sense to use it here as well.

Reply:
I don't want to use the `data_encoding_macro`, which introduces so much dependencies.

Reply:
Though you wouldn't need the `lazy_static` dependency. The dependencies `data_encoding_macro` introduces are well established ones though, that you'd need as soon as you use macros.

Reply:
The dependency `data_encoding_macro` uses syn 0.15 and I don't want to introduce it into my dependency graph. I will open a PR for `data-encoding`, and once it is merged, I will consider using the `data_encoding_marco`

Reply:
Not introducing a large dependency tree is surely reasonable. I'm OK with taking your current version. We can introduce `data_encoding_macro` later on if we like.

Reply:
Thanks for your review, if `data_encoding_macro` releases a new version, I will use it.

Reply:
@koushiro: it seems that `data_encoding_macro` was already updated to syn 1.0. But there wasn't a release yet.

Reply:
Yeah, I found it :(

Reply:
The `data-encoding-macro` v0.1.8 has been released, so I used it instead of `lazy_static`.